library/Fission/Web/Heroku.hs

```diff
@@ -8,20 +8,33 @@ import           Servant
 
 import           Fission.Prelude
 
+import qualified Fission.User                  as User
+import qualified Fission.User.CID              as User.CID
+import qualified Fission.Platform.Heroku.AddOn as Heroku.AddOn
+
 import qualified Fission.Web.Heroku.Provision   as Provision
 import qualified Fission.Web.Heroku.Deprovision as Deprovision
 import           Fission.Web.Server.Reflective
 
 type API = Provision.API :<|> Deprovision.API
 
 server ::
-  ( MonadDB               m
-  , MonadTime             m
-  , MonadThrow            m
-  , MonadLogger           m
-  , MonadLocalIPFS        m
-  , MonadRemoteIPFS       m
-  , MonadReflectiveServer m
+  ( MonadReflectiveServer    m
+  , MonadRemoteIPFS          m
+  , MonadLocalIPFS           m
+  , MonadLogger              m
+  , MonadLogger            t
+  , MonadThrow               m
+  , MonadThrow             t
```

Comment:
Keeping same constraints together. Lemme know if y'all would prefer to have the `t`s grouped instead.

Reply:
I do think it makes more sense to group the `t`s. but don't have a strong opinion on this at all